script/PSQL.py

```diff
@@ -0,0 +1,25 @@
+import psycopg2
+import datetime
+
+# user = "postgres"
+# password = "sL6IpcVZKv57HcZZTX3x"
```

Comment:
Each of these parameters should be coming from the environment (e.g. `os.environ['DB_PASSWORD']`)